packages/web3/src/token/index.test.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+// eslint-disable-next-line header/header
+import * as IndexExports from './index'
+import * as NFTModule from './nft'
+
+describe('index.ts exports', () => {
```

Comment:
I don't think this test is needed. The only thing in `index.ts` is  `export * from './nft'`